src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixPlugin.groovy

```diff
@@ -50,8 +50,16 @@ class ScalafixPlugin implements Plugin<Project> {
         project.sourceSets.each { SourceSet sourceSet ->
             if (ScalaSourceSet.isScalaSourceSet(project, sourceSet) && !extension.ignoreSourceSets.get().contains(sourceSet.name)) {
                 def scalaSourceSet = new ScalaSourceSet(project, sourceSet)
-                configureTaskForSourceSet(project, scalaSourceSet, IN_PLACE, fixTask, extension)
-                configureTaskForSourceSet(project, scalaSourceSet, CHECK, checkTask, extension)
+                ConfigureSemanticdbCompiler configureSemanticdbCompilerTask = project.tasks.create(
+                        "configure${sourceSet.name.capitalize()}SemanticdbCompilerPlugin", ConfigureSemanticdbCompiler) { task ->
```

Comment:
- `"configureSemanticdb${sourceSet.name.capitalize()}"`?
- let's add a group/description to it